src/components/list/list.js

```diff
@@ -190,20 +182,12 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
           secondaryItem = buttonWrapper[0];
         }
 
-        // Check for a secondary item and move it outside
-        if ( secondaryItem && (
-                secondaryItem.hasAttribute('ng-click') ||
-                ( tAttrs.ngClick &&
-                isProxiedElement(secondaryItem) )
-            )) {
-          // When using multiple secondary items we need to remove their secondary class to be
-          // orderd correctly in the list-item
-          if (hasSecondaryItemsWrapper) {
-            secondaryItem.classList.remove('md-secondary');
-          }
-          tEl.addClass('md-with-secondary');
-          container.append(secondaryItem);
-        }
+        // When using multiple secondary items we need to remove their secondary class to be
+        // orderd correctly in the list-item
+        secondaryItem.classList.remove('md-secondary');
```

Comment:
Just rechecked that code snippet, because you mentioned me on the IE10/9 issues with `classList` and figured out, that there's no need to remove that class anymore, because we are now always using a secondary container with static children. :+1: